app/components/GetProducts/CustomInput.tsx

```diff
@@ -0,0 +1,10 @@
+export default function CustomInput({ onChange, placeholder }) {
+    return (
+        <input
+            className="w-40 h-10 border pl-2 border-black mx-2"
+            type="text"
+            onChange={onChange}
+            placeholder={placeholder}
+        />
```

Comment:
Could add the required attribute here for required fields to prevent submission of form.

Reference: [required_attribute](https://www.w3schools.com/tags/att_input_required.asp)